events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+                        case "Trivia Choice Text Yellow":
+                            choiceTexts[1] = element;
+                            break;
+                        case "Trivia Choice Text Red":
+                            choiceTexts[2] = element;
+                            break;
+                        case "Trivia Choice Text Blue":
+                            choiceTexts[3] = element;
+                            break;
+                        case "Trivia Timer Countdown":
+                            timer = element;
+                            break;
+                        case "Trivia Avatar Counter Total":
+                            avatarCounter = element;
+                            break;
+                        case "Trivia Player Game Zone":
+                            gameZone = element;
+                            gameZoneProperties = Entities.getEntityProperties(gameZone, 
+                                ["id", "position", "dimensions", "rotation", "userData"]);
+                            break;
+                        case "Trivia Prize Amount":
+                            prizeDisplay = element;
+                            break;                            
+                        case "Trivia Bubble":
+                            bubble = element;
+                            break;
+                    }
+                }
+            }
+        });
+    }
+
+    function onClicked() { 
+        if (open) {
+            tablet.gotoHomeScreen();
+        } else {
+            tablet.gotoWebScreen(appPage);
+        }
+    }
+
+    // function getQuestion() {
+    //     Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEXT_QUESTION_SFX']);
+    //     try {
+    //         var triviaURL = SECRETS.trivia_URL,
+            
+    //         // if (type) {
+    //         //     triviaURL = triviaURL + "&type=" + type;
+    //         // }
+    //         // if (difficulty) {
+    //         //     triviaURL = triviaURL + "&difficulty=" + difficulty;
+    //         // }
+    //         // if (category) {
+    //         //     triviaURL = triviaURL + "&category=" + category;
+    //         // }
+    //         triviaURL = triviaURL + "category=" + "Giraffage_Hemsworth";
+    //         request(triviaURL, function (error, data) {
+    //             if (!error) {
+    //                 console.log(JSON.stringify(data));
+    //                 tablet.emitScriptEvent(JSON.stringify(data));
+    //                 triviaData = data;
+    //             }
+    //         });
+    //     } catch (err) {
+    //         console.log("err:", err);
+    //         print("Could not get domain data using userData domainAPIURL");
+    //     }
+    // }
+
+    function getQuestion() {
+        Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEXT_QUESTION_SFX']);
+        try {
+            var triviaURL = "https://opentdb.com/api.php?amount=1";
```

Comment:
are there any limitations to this API? rate limit, etc?

Reply:
Not that we have yet encountered.

Reply:
hehe, i'll take it